### PR TITLE
feat: add zkevm

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

File: src/constants/networks/zkevm.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+import EthereumLogo from 'assets/images/ethereum-logo.png'
+import { KS_SETTING_API } from 'constants/env'
+import { EVMNetworkInfo } from 'constants/networks/type'
+
+const EMPTY = ''
+const EMPTY_ARRAY: any[] = []
+const NOT_SUPPORT = null
+
+const zkEvm: EVMNetworkInfo = {
```
Comment (namgold) [inline src/constants/networks/zkevm.ts:11]: I saw they use `pze` for short of this chain

Comment (viet-nv) [reply src/constants/networks/zkevm.ts:11]: kệ họ

File: src/constants/networks/zkevm.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    routers: '0xF9c2b5746c946EF883ab2660BbbB1f10A5bdeAb4',
+    farms: ['0x86844C008F8799Ed0F067b3Dd886ea433Cc5e37c'],
+    farmv2Quoter: '0x6AFeb9EDd6Cf44fA8E89b1eee28284e6dD7705C8',
+    farmV2S: ['0xbD07Eaa92C701810df78eAa4e1c697a0206405E0'],
+  },
+  limitOrder: {
+    production: NOT_SUPPORT,
+    development: NOT_SUPPORT,
+  },
+  averageBlockTimeInSeconds: 2, // TODO: check these info
```
Comment (nguyenhoaidanh) [inline src/constants/networks/zkevm.ts:68]: todo

File: src/hooks/useKyberSwapConfig.ts [hunk 3]
```diff
@@ -65,7 +65,8 @@ const parseGlobalResponse = (
   chainId: ChainId,
 ): KyberswapGlobalConfig => {
   const data = responseData?.data?.config
-  const aggregatorDomain = data?.aggregator ?? AGGREGATOR_API
+  // TODO(viet-nv): revert
```
Comment (nguyenhoaidanh) [inline src/hooks/useKyberSwapConfig.ts:68]: todo

File: .env.adpr [hunk 4]
```diff
@@ -1,45 +1,44 @@
-VITE_AGGREGATOR_API=https://aggregator-api.dev.kyberengineering.io
-VITE_AGGREGATOR_STATS_API=https://aggregator-stats.dev.kyberengineering.io
```
Comment (nguyenhoaidanh) [inline .env.adpr:2]: todo

**Comment by nguyenhoaidanh** [comment]
lgtm now

File: src/constants/networks/zkevm.ts (hunk 2 above)
Comment (viet-nv) [reply src/constants/networks/zkevm.ts:68]: todo for all chains, still using in some classic farm. need to refactor soon

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**